src/components/functionElements/DraggableSwitch.vue

```diff
@@ -18,6 +20,22 @@ export default {
           : true;
       },
     },
+    styleType: {
+      type: String,
+      validator(value) {
+        return value
+          ? ['checkbox'].includes(value)
+          : true;
+      },
+    },
+    size: {
+      type: String,
+      validator(value) {
+        return value
+          ? ['small'].includes(value)
+          : true;
```

Comment:
this styling looks nicer
can you check in other places and make this ternary in validators the same about styling (multiline I mean)